lib/igniter/code/common.ex

```diff
@@ -24,6 +24,36 @@ defmodule Igniter.Code.Common do
     end
   end
 
+  @doc """
+  Moves to the last node that matches the predicate.
+
+  Similar to `move_to/2` but it doesn't stop at the first match,
+  for example a zipper for the following code:
+
+  ```elixir
+  port = 4000
+  port = 4001
+  ```
+
+  With a match for `port = _` as `{:=, _, [{:port, _, _}, _]}`,
+  will return the second `port` variable.
+  """
+  @spec move_to_last(Zipper.t(), (Zipper.t() -> boolean())) :: {:ok, Zipper.t()} | :error
+  def move_to_last(zipper, pred) when is_function(pred, 1) do
```

Comment:
Added this function since this pattern seems common enough to be exposed. I have other use cases where an `:after` option would be useful and use `move_to_last/2` too.